src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/ArgumentValueResolverPass.php

```diff
@@ -20,7 +20,7 @@
  *
  * @author Iltar van der Berg <kjarli@gmail.com>
  */
-class ControllerArgumentValueResolverPass implements CompilerPassInterface
+class ArgumentValueResolverPass implements CompilerPassInterface
```

Comment:
This rename looks wrong to me. Compiler passes is in the `Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler` namespace, which gives no hints as to what Argument refers to.